docs/src/main/paradox/lagom/sample-applications.md

```diff
@@ -0,0 +1,15 @@
+# Lagom gRCP sample applications
+
+There are also samples demonstrating the use of gRPC in Lagom in both Java and Scala.
+
+## Java
+
+- Guide: <https://developer.lightbend.com/guides/lagom-java-grpc-example/>
+- Download: <https://developer.lightbend.com/start/?group=lagom&project=lagom-java-grpc-example>
+- Github: <https://github.com/lagom/lagom-java-grpc-example>
+
+## Scala
+
+- Guide: <https://developer.lightbend.com/guides/lagom-scala-grpc-example/>
+- Download: <https://developer.lightbend.com/start/?group=lagom&project=lagom-scala-grpc-example>
+- Github: <https://github.com/lagom/lagom-scala-grpc-example>
```

Comment:
Should these be paradox-grouped?

Reply:
Alls the dev.lb.com links are broken 😅 (I know you've created PRs separately to fix it)

Reply:
>  Should these be paradox-grouped?

I don't think so. There is no further information on the page, so I don't think we need `@@@ div { .group-scala }` here. It is a pretty small list too so I would leave it as it is now.